background.ts

```diff
@@ -84,6 +84,16 @@ Electron.app.whenReady().then(async() => {
     }
 
     buildApplicationMenu();
+    if (os.platform().startsWith('win')) {
+      const appVersion = await getVersion();
+
+      Electron.app.setAboutPanelOptions({
+        copyright:          'Copyright © 2021 SUSE LLC', // TODO: Update this to 2021-... as dev progresses
```

Comment:
It would be good to set it at build time rather than requiring somebody to update things manually.

Reply:
What if we hit a year where no actual development is done on the app, but a build is run?

Originally, I was going to set the copyright to `2021` during this year and `2021 - ${ new Date().getFullYear() }` otherwise. And then I realized just because someone is running this in 2030 doesn't necessarily mean the copyright extends to that year. And then after some time, it's possible that RD automatically enters the public domain. So no, this shouldn't be left up to software. Better to put up with a semi-permanent TODO marker to jog a human's memory.

Reply:
Right, _ideally_ it would be from the date of the last commit.

… Yeah we're probably not going to expend that much effort.